contract/services.py

```diff
@@ -0,0 +1,43 @@
+import core
+
+
+@core.comparable
+class PolicyHolderContract(object):
+
+    def __init__(self, policy_holder):
+        self.policy_holder = policy_holder
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
+
+    def submit(self, submit):
+        pass
+
+    def amend(self, submit):
+        pass
+
+
+@core.comparable
+class PolicyHolderContractDetails(object):
+
+    def __init__(self, policy_holder):
+        self.policy_holder = policy_holder
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
+
+
+@core.comparable
+class Payment(object):
+
+    def __init__(self, policy_holder):
+        self.policy_holder = policy_holder
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
+
```

Comment:
please add, create, update, delete and asign creditnote

Reply:
Methods added